virttest/libvirt_xml/vol_xml.py

```diff
@@ -31,7 +31,7 @@ class VolXMLBase(base.LibvirtXMLBase):
 
     __slots__ = ('name', 'key', 'capacity', 'allocation', 'format', 'path',
                  'owner', 'group', 'mode', 'label', 'compat', 'lazy_refcounts',
-                 'encryption', "capacity_unit")
+                 'encryption', 'capacity_unit', 'clusterSize', 'clusterSize_unit')
```

Comment:
Add explanation about those 2 newly added parameter after line 30

Reply:
Sorry for forgetting it and now it has already been added.